js/Track.js

```diff
@@ -402,6 +402,11 @@ const Track = Base.extend({
       height = Math.max(height, this.prop('minLabelHeight'));
     }
 
+    if (this.prop("minHeight")) {
+      var minHeight = parseInt(this.prop("minHeight"))
```

Comment:
If we introduce new parameter could we add it top of parameters & leave comment what its doing.